src/util/LineChart/MiniGraph.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="miniGraph">
+    <span v-if="data.length === 1" class="miniGraph__noGraph">{{
+      $t('kpi.noGraph')
+    }}</span>
+    <svg
+      v-else
+      width="100%"
+      height="100%"
+      viewBox="0 0 800 200"
+      preserveAspectRatio="xMidYMid meet"
+    >
+      <g class="lineChart" :transform="translate">
+        <path class="area" :d="area" />
+        <path class="line" :d="line" />
+      </g>
+    </svg>
+  </div>
+</template>
+
+<script>
+import * as d3 from 'd3';
```

Comment:
Kan vi ikke importere kun de tingene vi trenger fra d3 slik som vi pleier å gjøre ellers?

For eksempel `import { scaleTime, scaleLinear } from 'd3-scale';` osv.